addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -23,11 +25,29 @@ export class Attachment extends FileModelMixin(Record) {
     thread = fields.One("Thread", { inverse: "attachments" });
     /** @type {string} */
     raw_access_token;
+    /** @type {string} */
+    thumbnail_access_token;
```

Comment:
Also alphabetical (as good as possible, the file is a mess...). Either together with other "t" fields, or at least after all "r" fields, but not in the middle of them.